package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/exaptis/asciidoctor-loader",
   "dependencies": {
-    "asciidoctor.js": "~1.5.5-2",
+    "asciidoctor.js": "1.5.5-4",
```

Comment:
I recommend using a fix version because Asciidoctor.js does not comply with semver.
The reason is that we want to use the same version number between Asciidoctor (Ruby), Asciidoctor.js (JavaScript), AsciidoctorJ (Java).